tests/test_examples.py

```diff
@@ -386,38 +304,6 @@ def test_ATF_results(tmpdir_factory):
     np.testing.assert_allclose(theta_err, 0, atol=5e-4)
 
 
-@pytest.mark.regression
-@pytest.mark.solve
-def test_ESTELL_results(tmpdir_factory):
```

Comment:
This is basically testing the same thing as the ATF test and all of the heliotron/dshape/solovev ones, but this one takes the longest by far.